tempesta_fw/http_parser.c

```diff
@@ -598,14 +599,12 @@ good_looking_eol:							\
 		case 0xdada:						\
```

Comment:
In line 545: there is no need to do `& 0xf` since you accept 0xd and 0xa only in the IF statement.